lib/src/dash/better_player_dash_utils.dart

```diff
@@ -0,0 +1,144 @@
+// Dart imports:
+import 'dart:convert';
+import 'dart:io';
+
+// External Package imports:
+import 'package:better_player/src/hls/hls_parser/mime_types.dart';
+import 'package:xml/xml.dart';
+
+// Package imports:
+import 'package:better_player/src/core/better_player_utils.dart';
+import 'package:better_player/src/dash/better_player_dash_audio_track.dart';
+
+// Project imports:
+import 'package:better_player/src/dash/better_player_dash_subtitle.dart';
+import 'package:better_player/src/dash/better_player_dash_track.dart';
+
+import 'better_player_dash_video.dart';
+
+class DashObject {
+  List<BetterPlayerDashVideo>? videos;
+  List<BetterPlayerDashSubtitle>? subtitles;
+  List<BetterPlayerDashAudioTrack>? audios;
+
+  DashObject({this.videos, this.subtitles, this.audios});
+}
+
+///DASH helper class
+class BetterPlayerDashUtils {
```

Comment:
Seems like some duplication from Hls utils. Maybe you could create share common logic between these utils?

Reply:
Solved in c819369